moto/ecs/models.py

```diff
@@ -1327,6 +1417,134 @@ def untag_resource(self, resource_arn, tag_keys):
                 raise ServiceNotFoundException(service_name=parsed_arn["id"])
         raise NotImplementedError()
 
+    def create_task_set(
+        self,
+        service,
+        cluster,
+        task_definition,
+        external_id=None,
+        network_configuration=None,
+        load_balancers=None,
+        service_registries=None,
+        launch_type=None,
+        capacity_provider_strategy=None,
+        platform_version=None,
+        scale=None,
+        client_token=None,
+        tags=None,
+    ):
+        task_set = TaskSet(
```

Comment:
And definitely named variables here - very easy to mess up the order of such a big list of args